qutebrowser/misc/sessions.py

```diff
@@ -19,23 +19,20 @@
 
 """Management of sessions - saved tabs/windows."""
 
+import itertools
 import os
 import os.path
-import itertools
 import urllib
 
-from PyQt5.QtCore import QUrl, QObject, QPoint, QTimer
-from PyQt5.QtWidgets import QApplication
 import yaml
-
-from qutebrowser.utils import (standarddir, objreg, qtutils, log, message,
-                               utils)
+from PyQt5.QtCore import QObject, QPoint, QTimer, QUrl
+from PyQt5.QtWidgets import QApplication
 from qutebrowser.commands import cmdexc, cmdutils
```

Comment:
Should be a blank line between third party and qutebrowser imparts.

Reply:
There is now :)